libavcodec/cbs.c

```diff
@@ -158,7 +158,7 @@ av_cold void ff_cbs_close(CodedBitstreamContext **ctx_ptr)
     av_freep(ctx_ptr);
 }
 
-static void cbs_unit_uninit(CodedBitstreamUnit *unit)
+void ff_cbs_unit_uninit(CodedBitstreamUnit *unit)
```

Comment:
why do we need this? better give some explain on the commit log

Reply:
Sure.

Reply:
Will it be better to change it to ff_cbs_unref_unit?

Reply:
I mean, why we need to expose this to public. 
can we just take a reference to unit->content_ref, like slice_start

Reply:
Goot catch. I use the same way to add a reference to the unit content_ref and remove the redundant commit for exposing ref unit.